mentoring/templates/html/mentoring.html

```diff
@@ -1,16 +1,44 @@
-<div class="mentoring">
+<div class="mentoring" data-mode="{{ self.mode }}" data-step="{{ self.step }}">
   <div class="missing-dependency warning" data-missing="{{ self.has_missing_dependency }}">
     You need to complete <a href="{{ missing_dependency_url }}">the previous step</a> before
     attempting this step.
   </div>
-  {% for name, c in named_children %}
-  {{c.body_html|safe}}
-  {% endfor %}
-  {% if self.display_submit %}
-  <input type="button" value="submit" class="submit"></input>
-  <span class="progress" data-completed="{{ self.completed }}">
-    <span class='indicator'></span>
-  </span>
+
+  {% if title %}
+  <div class="title">
+    {% if title %} <h2>{{ title }}</h2> {% endif %}
+  </div>
   {% endif %}
-  <div class="messages"></div>
+
+  <div class="{{self.mode}}-question-block">
+    {{child_content|safe}}
+
+    {% if self.display_submit %}
```

Comment:
@antoviaque I actually noticed that too. As far as I could tell after looking into it for a while, it's some sort of weird JavaScript error caused by RequireJS or Discussion XBlock or something. Will continue to investigate tomorrow. 

I tried logging into that sandbox to check the JS console and see if it's the same error I saw, but it says the staff@example.com account has not been activated.